observation_portal/accounts/serializers.py

```diff
@@ -105,3 +117,126 @@ class RevokeTokenResponseSerializer(serializers.Serializer):
 
 class AcceptTermsResponseSerializer(serializers.Serializer):
     message = serializers.CharField(default='Terms accepted')
+
+
+class CreateUserSerializer(serializers.Serializer):
+    """Flattend User/Profile model seralizer with the bare minimum needed to
+    create an account.
+    """
+
+    # User model stuff
+    username = serializers.CharField(
+        max_length=150,
+        validators=[
+            validators.UniqueValidator(
+                queryset=User.objects.all(),
+                message="username already exists",
+                lookup="exact"
+            )
+        ]
+    )
+
+    # allow passing in the password, but if one is not provided generate one
+    # also don't return this in the response (write only)
+    password = serializers.CharField(
+        max_length=128, required=False, default=None, write_only=True
+    )
+    first_name = serializers.CharField(max_length=150)
+    last_name = serializers.CharField(max_length=150)
+    email = serializers.EmailField(
+        validators=[
+            validators.UniqueValidator(
+                queryset=User.objects.all(),
+                message="user with email already exists",
+                lookup="iexact"
+            )
+        ]
+    )
+
+    # Profile model stuff
+    institution = serializers.CharField(max_length=200)
+    title = serializers.CharField(max_length=200)
+    education_user = serializers.BooleanField(required=False, default=True)
+
+
+class BulkCreateUsersSerializer(serializers.Serializer):
+    users = serializers.ListField(child=CreateUserSerializer(), max_length=50)
```

Comment:
I think the serializer class should be dynamically loaded, so it can be overwritten like the other serializers.